Source/Core/DolphinQt/Config/GeckoCodeWidget.cpp

```diff
@@ -165,7 +165,7 @@ void GeckoCodeWidget::OnSelectionChanged()
   m_code_view->clear();
 
   for (const auto& c : code.codes)
```

Comment:
I realize this is pre-existing code, but would you mind adding curlies to this; since the contents of the `for` loop spans over multiple lines?

Reply:
done